larpandoracontent/LArThreeDReco/LArTransverseTrackMatching/ClearTracksTool.cc

```diff
@@ -24,14 +24,14 @@ ClearTracksTool::ClearTracksTool() :
 
 bool ClearTracksTool::Run(ThreeViewTransverseTracksAlgorithm *const pAlgorithm, TensorType &overlapTensor)
 {
-    if (PandoraContentApi::GetSettings(*pAlgorithm)->ShouldDisplayAlgorithmInfo())
-    std::cout << "----> Running Algorithm Tool: " << this->GetInstanceName() << ", " << this->GetType() << std::endl;
-
     bool particlesMade(false);
+    if (PandoraContentApi::GetSettings(*pAlgorithm)->ShouldDisplayAlgorithmInfo())
+    {
 
     TensorType::ElementList elementList;
     overlapTensor.GetUnambiguousElements(true, elementList);
     this->CreateThreeDParticles(pAlgorithm, elementList, particlesMade);
+    }
```

Comment:
This modification is not correct. The conditional has been expanded to encompass code that should not be conditional.